libsolidity/ast/Types.cpp

```diff
@@ -968,9 +976,18 @@ BoolResult RationalNumberType::isExplicitlyConvertibleTo(Type const& _convertTo)
 	auto category = _convertTo.category();
 	if (category == Category::FixedBytes)
 		return false;
-	if (category == Category::Address)
+	else if (category == Category::Address)
+	{
 		if (isNegative() || isFractional() || integerType()->numBits() > 160)
 			return false;
+	}
+	else if (category == Category::Integer)
```

Comment:
Can't you just return false here because we said that explicit and implicit conversions should be the same?

Reply:
I changed it to return false.